scripts/weaviate-test-helpers/create_schema.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
```

Comment:
Is there a more standard way to define a weaviate schema that we could leverage for documentation that wouldn't require python?